tools/styles/tokens/spectrum/system-theme-bridge.css

```diff
@@ -409,6 +409,24 @@
     --system-button-static-black-secondary-outline-background-color-focus: var(
         --spectrum-transparent-black-300
     );
+    --system-button-group-spacing-horizontal: var(--spectrum-spacing-300);
+    --system-button-group-spacing-vertical: var(--spectrum-spacing-300);
+    --system-button-group-size-s-spacing-horizontal: var(
+        --spectrum-spacing-200
+    );
+    --system-button-group-size-s-spacing-vertical: var(--spectrum-spacing-200);
+    --system-button-group-size-m-spacing-horizontal: var(
+        --spectrum-spacing-300
+    );
+    --system-button-group-size-m-spacing-vertical: var(--spectrum-spacing-300);
+    --system-button-group-size-l-spacing-horizontal: var(
+        --spectrum-spacing-300
+    );
+    --system-button-group-size-l-spacing-vertical: var(--spectrum-spacing-300);
+    --system-button-group-size-xl-spacing-horizontal: var(
+        --spectrum-spacing-300
+    );
+    --system-button-group-size-xl-spacing-vertical: var(--spectrum-spacing-300);
```

Comment:
@rise-erpelding This came in with `14.1.6`.

Reply:
That seems unexpected as this is related to the buttongroup package which has not been changed here. Pulling the branch down locally and running another `yarn build` I'm seeing these changes removed. Wondering if the same would be true for you?